backend/app/api/endpoints/kind/skills.py

```
@@ -1029,7 +1041,36 @@ def download_skill(
+        from app.models.task import TaskResource
+        from app.services.task_member_service import task_member_service
+
+        # Verify current user is a member of the task (owner or group member)
+        if task_member_service.is_member(db, task_id, current_user.id):
+            # Get task to find the owner
+            task = (
+                db.query(TaskResource)
+                .filter(
+                    TaskResource.id == task_id,
+                    TaskResource.kind == "Task",
+                    TaskResource.is_active == True,
```

Ruff (0.14.14)

[error] Avoid equality comparisons to `True`; use `TaskResource.is_active:` for truth checks

Replace with `TaskResource.is_active`

(E712)